modules/KalturaSupport/components/largePlayBtn.js

```diff
@@ -101,6 +101,9 @@
 							.on(eventName, function(e) {
 								_this.clickButton(e);
 							} );
+				if (!mw.isWindowsPhone()){
+					this.$el.css("border-radius", "0.1em");
```

Comment:
We should add it as css and not hardcoded - think of someone doing their own skin....